developer/src/tike/child/UfrmDebug.pas

```diff
@@ -894,7 +900,8 @@     TMemoSelectionState = record
     state: TMemoSelectionState;
   begin
     state := SaveMemoSelectionState;
-    memo.SelText := Text;
+    // Line breaks: replace \n with \r\n so line breaks work
+    memo.SelText := ReplaceStr(Text, #$0D, #$0D#$0A);
```

Comment:
What keeps this from turning `#$0D#$0A` into `#$0D#$0D#$0A` (and so forth)?  And isn't this replacing `\r`, not `\n`?

Reply:
> What keeps this from turning `#$0D#$0A` into `#$0D#$0D#$0A` (and so forth)? 

The `Text` parameter will never have `#$0D#$0A`; it will only ever have `#$0D`.

> And isn't this replacing `\r`, not `\n`?

Yes, right.